openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1077,70 +1098,87 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
-* `c_nonce`: OPTIONAL. The `c_nonce` as defined in (#credential-response). 
-* `c_nonce_expires_in`: OPTIONAL. The `c_nonce_expires_in` as defined in (#credential-response).
+The Batch Credential Response may be encrypted if the `credential_response_encryption` object was present in the request, see (#credential-response) for details.
+If the Credential Response is not encrypted, the media type of the response MUST be set to `application/json`.
 
-Below is a non-normative example of a Batch Credential Response in an immediate issuance flow:
+The following parameters are used in the JSON-encoded Batch Credential Response body:
 
-```
-HTTP/1.1 200 OK
-Content-Type: application/json
-Cache-Control: no-store
+* `credential_responses`: REQUIRED. Array that contains all Batch Credential Response objects. Each element within the array matches the corresponding Batch Credential Request object by array index in the `credential_requests` parameter of the Batch Credential Request. A Batch Credential Response object contains either a `credentials` object (and possibly a `notification_id`) or a `transaction_id` parameter. Each element may refer to a different Credential Configuration or Credential Dataset.
+  * `credentials`: REQUIRED if `transaction_id` is not present within a particular Batch Credential Response object. It contains an array of issued Credentials, these represent specific instances of a Credential with the same Credential Configuration and Credential Dataset but with different cryptographic material. It MAY be a string or an object, depending on the Credential format. See (#format-profiles) for the Credential format specific encoding requirements.
+  * `notification_id`: OPTIONAL. String identifying a batch of issued Credentials that the Wallet includes in the Notification Request as defined in (#notification). This parameter MUST NOT be present if `credential` parameter is not present.
+  * `transaction_id`: REQUIRED if `credentials` is not present within a particular Batch Credential Response object. It contains a String identifying a Deferred Issuance transaction. This claim is contained in the response if the Credential Issuer was unable to immediately issue the Credential. The value is subsequently used to obtain the respective Credential with the Deferred Credential Endpoint (see (#deferred-credential-issuance)). It MUST be invalidated after the Credential for which it was meant has been obtained by the Wallet.
+* `c_nonce`: OPTIONAL. The `c_nonce` as defined in (#credential-response).
+* `c_nonce_expires_in`: OPTIONAL. The `c_nonce_expires_in` as defined in (#credential-response).
```

Comment:
again, why is there a need to redefine the parameters here instead of saying all the parameters defined in (#credential-response) other than `credential` need to be used?

Reply:
applied simplifications, please recheck